tests/XRootD/CMakeLists.txt

```diff
@@ -34,10 +40,22 @@ foreach(CONFIG ${XROOTD_CONFIGS})
   list(APPEND ${CONFIG}_LOGS ${CMAKE_CURRENT_BINARY_DIR}/${CONFIG}/client.log)
   list(APPEND ${CONFIG}_LOGS ${CMAKE_CURRENT_BINARY_DIR}/${CONFIG}/xrootd.log)
 
+  list(APPEND ${CONFIG}_FIXTURES XRootD::${CONFIG})
+  if(${CONFIG} STREQUAL "scitokens")
+    list(APPEND ${CONFIG}_FIXTURES SciTokens)
+  endif()
```

Comment:
Please do this outside of the `foreach` loop. If statements inside a for loop when they are only true once is not a good pattern.

Reply:
Fixed in latest rebase.